srcpkgs/dmd2.091/template

```diff
@@ -0,0 +1,100 @@
+# Template file for 'dmd2.091'
```

Comment:
If it's only going to be used for bootstrap, could we simplify the package with fewer subpackages and rm'ing everything that's not important?

Reply:
Yes, as pointed out by @sgn, I should probably just get rid of everything non-essential. I should probably trim the `dmd2.081`  package as well, although in a different PR.